dbos/_dbos_config.py

```diff
@@ -407,6 +407,7 @@ def configure_db_engine_parameters(
         "pool_timeout": 30,
         "max_overflow": 0,
         "pool_size": 20,
+        "pool_pre_ping": True,
```

Comment:
Not sure if we want to set it to True by default. This does add a bit of overhead whenever we get a connection from the pool. It might be okay if we document this carefully.

Reply:
We've seen enough users have trouble with stale connections that I think it's worth it to fix them by default